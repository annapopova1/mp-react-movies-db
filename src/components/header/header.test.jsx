import React from 'react';
import Header from './header';

describe('<Header/>', () => {
  test('should render Header component for main page', () => {
    const header = shallow(<Header />);
    expect(header).toMatchSnapshot();
  });

  test('should render Header component for secondary page', () => {
    const header = shallow(<Header isMainPage={false} />);
    expect(header).toMatchSnapshot();
  });
});
