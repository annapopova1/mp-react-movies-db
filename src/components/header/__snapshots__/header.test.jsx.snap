// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header/> should render Header component for main page 1`] = `
<header
  className="container navbar navbar-dark bg-dark py-1"
>
  <span
    className="navbar-brand"
  >
    Movies DB
  </span>
</header>
`;

exports[`<Header/> should render Header component for secondary page 1`] = `
<header
  className="container navbar navbar-dark bg-dark py-1"
>
  <span
    className="navbar-brand"
  >
    Movies DB
  </span>
  <button
    className="btn btn-outline-success"
    onClick={[Function]}
    type="submit"
  >
    SEARCH
  </button>
</header>
`;
